src/pages/CarDetails.js

```diff
@@ -1,3 +1,36 @@
-const CarDetails = () => <h1>Car details page</h1>;
+import React from 'react';
+import carImageBig from '../images/car-big.png';
+
+const CarDetails = () => (
+  <div className="flex flex-col gap-[30%] p-2 items-center justify-center">
```

Comment:
For the main div element, kindly add `md:w-4/5`, so that the sidebar is more consistent

Reply:
Alright...thank you